### PR TITLE
@bruno/more android fixes

### PR DESCRIPTION
**Pull request by brunobar79**
- Setup fastlane for android
- Change savings APYPill font color to green
- Prevent fetching backup password on android which triggers an error because it's an iOS only feature.
- Fix import that was causing Keychain Integrity checks to trigger a false positive
- Fixes RAI-922  - Added back button to settings title
- Fixes RAI-938 - Edit individual wallet button not tappable

### CODE REVIEW

**Review by skylarbarrera — COMMENTED** [review]
settings back button doesnt show on Pixel: https://cloud.skylarbarrera.com/Screenshot_20201113-224459.png
also I think we should use an 'x' vs back arrow, on headerRight

rest of the stuff looks good, cant speak on fastlane

**Comment by brunobar79** [comment]
@skylarbarrera  I tested on a pixel and it was there. Can u try again and make sure you're connected to the packager?

File: src/screens/SettingsModal.js [hunk 1]
```diff
@@ -154,7 +157,7 @@ export default function SettingsModal() {
 
   return (
     <Modal
-      fullScreenOnAndroid
+      fullScreenOnAndroid={false}
```
Comment (skylarbarrera) [inline src/screens/SettingsModal.js:160]:
```suggestion
      fullScreenOnAndroid
```

**Review by skylarbarrera — APPROVED** [review]
looks good, added a change to fix the following: https://cloud.skylarbarrera.com/Screenshot_20201114-203846.png

unless we want to change settings to be similar to the iOS modal we should keep fullscreen

**Review by osdnk — APPROVED** [review]

**Pull request merged**